fili-core/src/main/java/com/yahoo/bard/webservice/druid/client/impl/AsyncDruidWebServiceImpl.java

```diff
@@ -57,30 +58,70 @@
     private final Meter httpErrorMeter;
     private final Meter exceptionMeter;
     private static final MetricRegistry REGISTRY = MetricRegistryFactory.getRegistry();
+
     public static final String DRUID_TIMER = "DruidProcessing";
     public static final String DRUID_QUERY_TIMER = DRUID_TIMER + "_Q_";
     public static final String DRUID_QUERY_ALL_TIMER = DRUID_QUERY_TIMER + "All";
     public static final String DRUID_QUERY_MAX_TIMER = DRUID_QUERY_TIMER + "Max";
     public static final String DRUID_WEIGHTED_QUERY_TIMER = DRUID_TIMER + "_W_";
     public static final String DRUID_SEGMENT_METADATA_TIMER = DRUID_TIMER + "_S_0";
 
+    public static final Function<Response, JsonNode> DEFAULT_JSON_NODE_BUILDER_STRATEGY =
+            new Function<Response, JsonNode>() {
+
+        @Override
+        public JsonNode apply(final Response response) {
+            try {
+                return new MappingJsonFactory().createParser(response.getResponseBodyAsStream()).readValueAsTree();
+            } catch (IOException ioe) {
+                throw new IllegalStateException(ioe);
```

Comment:
Probably doesn't hurt, but since it's about to be thrown, isn't likely to buy us much when the parent catches and logs it.